### ISSUE TITLE
Does Wan2.2-Fun-Control support input_image conditioning or only reference_image?

### ISSUE THREAD

**Issue by LafouCC** [modelscope/DiffSynth-Studio#972]
Affected Models:
Wan2.2-Fun-A14B-Control (in_dim: 52)

When using Fun Control models (which have `require_clip_embedding: False`), even if y contains valid VAE-encoded image latents from `WanVideoUnit_ImageEmbedderVAE`, the code in `WanVideoUnit_FunControl`:

```
if clip_feature is None or y is None:
    clip_feature = torch.zeros((1, 257, 1280), dtype=pipe.torch_dtype, device=pipe.device)
    y = torch.zeros((1, y_dim, (num_frames - 1) // 4 + 1, height//8, width//8), dtype=pipe.torch_dtype, device=pipe.device)
else:
    y = y[:, -y_dim:]
y = torch.concat([control_latents, y], dim=1)
return {"clip_feature": clip_feature, "y": y}
```
overwrites y with zeros, discarding the input_image conditioning.

FUN models appear designed to accept image conditioning based on their in_dim values (16 video + 16 control + 16 image + optional 4 mask), but the current logic prevents this. Is this intentional design? Are Fun Control models meant to use `reference_image` as the sole method for first-frame conditioning, rather than input_image through the y? Or should the conditions be separated to allow y to retain image conditioning when clip_feature is None?

**Comment by Artiprocher** [comment]
@LafouCC This model doesn't support input_image. The code is designed for both I2V models and FUN models, thus it may look confusing.
